scripts/krad-conversion/src/test/resources/test.config.properties

```diff
@@ -31,6 +31,7 @@ project.artifact.version = "2.2.1"
 project.artifact.type="war"
 
 // experimental - use svn checkout rather than existing source
+// needs to be changed to git/github
 project.use.svn="false"
 project.svn.path="https://svn.kuali.org/repos/rice/tags/2.2.1/rice-sampleapp/"
```

Comment:
It's experimental feature to the conversion script so I wasn't going to worry about it in this release.